### PR TITLE
NEW support of Stripe Terminal with Takepos

### PR DESCRIPTION
**Pull request by ptibogxiv**
#19444 new PR
A recent change on Stripe could update with a better code later (less javascript)

### CODE REVIEW

File: htdocs/stripe/admin/stripe.php [hunk 1]
```diff
@@ -104,6 +104,10 @@
 	if (!$result > 0) {
 		$error++;
 	}
+	$result = dolibarr_set_const($db, "STRIPE_LOCATION", GETPOST('STRIPE_LOCATION', 'alpha'), 'chaine', 0, '', $conf->entity);
```
Comment (eldy) [inline htdocs/stripe/admin/stripe.php:107]: What is Stripe Location ?

Comment (eldy) [inline htdocs/stripe/admin/stripe.php:107]: You should add  
if (GETPOSTISSET('STRIPE_LOCATION'))
so when switching from debug and not debug, updating setup will not clear setup.

Comment (ptibogxiv) [reply htdocs/stripe/admin/stripe.php:108]: @eldy done today :)

**Comment by eldy** [comment]
@ptibogxiv Can you write a chapter on this wiki page https://wiki.dolibarr.org/index.php/Module_Stripe_FR
to explain how we can use the Stripe TPE ?
Setup to do on stripe side and dolibarr side. How to test it.

**Pull request merged**